common/lib/symbiosis/ssl/hooks.rb

```diff
@@ -19,9 +19,9 @@ def run!(event, domains)
         @event = event
         @domains = domains
 
-        Dir.glob(@hooks_dir)
-           .filter(valid_hook?)
-           .all?(runs_successfully?)
+        Dir.glob(@hooks_glob)
+           .select { |h| valid_hook?(h) }
+           .all? { |h| runs_successfully?(h) }
```

Comment:
arg, that's embarrassing. It's be nice if there was a more concise way to get a proc from a method than &method(:method_name_here) or what you've done - which honestly seems like the better of the two.